felix/fv/bpf_test.go

```diff
@@ -744,6 +752,24 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 
 			err := infra.AddDefaultDeny()
 			Expect(err).NotTo(HaveOccurred())
+
+			if !testOpts.connTimeEnabled {
```

Comment:
Any  tests that can have their `testOpts.connTimeEnabled` condition revoked now?

Reply:
This sets environment prerequisites. It is not necessary if felix sets it on its own. I moved it to felix itself. So when we enable BPF, felix sets all=0 heps=2.